tools/mqppep/.shed.yml

```diff
@@ -0,0 +1,29 @@
+name: mqppep
+owner: galaxyp
+description: MaxQuant Phosphoproteomic Enrichment Pipeline - Preprocessing and ANOVA
+homepage_url: 'https://github.com/galaxyproteomics/tools-galaxyp/'
+long_description: |
+  Phosphoproteomic Enrichment Pipeline provides tools two tools for processing MaxQuant data for phosphoproteome-enriched samples.
+ 
+  "MaxQuant Phosphopeptide Preprocessing" (mqppep_preproc) transforms the 'Phospho (STY)Sites.txt' output file that MaxQuant produces for for phosphoproteome-enriched samples to prepare it for ANOVA.
+
+  "MaxQuant Phosphopeptide ANOVA" (mqppep_anova) performs ANOVA on the preprocessed data.
+
+remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/mqppep
+type: unrestricted
+categories:
+- Proteomics
+
+suite:
+  name: "suite_mqppep"
+  description: suite for preprocessing, ANOVA, and KSEA of MaxQuant results for phosphoproteomic enrichment
+  long_description: |
+    Phosphoproteomic Enrichment Pipeline provides tools for preprocessing
+    the 'Phospho (ST)Sites.txt' or 'Phospho (Y)Sites.txt' files produced
+    by MaxQuant and for ANOVA analysis of the preprocessed data.
+
+repositories:
```

Comment:
why not removing this and using the standard approach of autogenerating everything from tool ID and tool description? Less duplication in the end.

Reply:
@bgruening Thank you for this suggestion.  Now lint passes without include!